plugins/modules/mssql_script.py

```diff
@@ -165,6 +191,27 @@
 """
 
 RETURN = r"""
+script_source:
+  description: Source of the executed script.
+  type: str
+  returned: always
```

Comment:
This isn't returned if the module fails. Also all new return values need a `version_added` entry.
```suggestion
  returned: success
  version_added: 11.0.0
```